task-management/src/main/webapp/vue-app/tasks-management/components/Project/AddProjectDrawer.vue

```diff
@@ -265,8 +265,8 @@ export default {
             this.manager.forEach(manager_el => {
               if (manager_el.providerId ==='space') {
                 projects.spaceName = eXo.env.portal.spaceName;
-                projects.manager.push(`manager:/spaces/${manager_el.remoteId}`);
-                projects.participator.push(`member:/spaces/${manager_el.remoteId}`);
+                projects.manager.push(`manager:/spaces/${eXo.env.portal.spaceGroup}`);
+                projects.participator.push(`member:/spaces/${eXo.env.portal.spaceGroup}`);
```

Comment:
please pay attention to this, we can display the Task application outside a space context which will lead to a null value for `eXo.env.portal.spaceGroup`. You will have to retrieve the group name of the space from space entity assigned to project itself.

Reply:
Thanks for this information